### ISSUE TITLE
Launching with Intel+Nouveau (PRIME): Multiple renderers at same time not supported

### ISSUE THREAD

**Issue by DawidLoubser** [swaywm/sway#2184]
Hi,

First of all, amazing work - especially with wlroots. You're doing the linux community an incredible service.

I'm currently running Awesome WM with Intel + Nouveau. I need the latter, because my laptop connects external displays via the nvidia card. I run all graphics on the intel processor though. With the open-source nouveau driven, it all "just works" - I see multiple outputs in xrandr, and can extend my display etc.

Does sway currently support something like this? i.e. launching on intel, but connecting external displays via a different card (nvidia)? When I attempt to run `sway` I simply get an error message stating that multiple renderers at the same time are not supported.

Happy to dig deep myself, but I'm new to this project, and any pointers would be appreciated.

**Comment by emersion** [comment]
It _should_ be supported, but it's broken at the moment.

cc @ascent12

**Comment by DawidLoubser** [comment]
Thanks @emersion ! I'm running latest Git master BTW.

I'll keep an eye on this then, with the hope that it gets fixed in the near future - with full respect to the complexities involved, and wishing I could help you but my C is pretty weak.

**Comment by ascent12** [comment]
Multi-GPU support is _supposed_ to be supported (i.e. it used to work), but is currently broken, waiting on myself to fix it. I've been dragging my feet a bit, just due to how annoying it is to test. It's something I hope to get around to doing before sway 1.0 is released, but it depends if I can muster up the time and motivation to do it.

>with full respect to the complexities involved, and wishing I could help you but my C is pretty weak

Yeah, normally we would encourage people to try and have a go at fixing bugs themselves, but this is **way** beyond what a new contributor could casually fix.

It's actually a wlroots problem instead of a sway problem, and is a covered by https://github.com/swaywm/wlroots/issues/934, so I'll just close this ticket as a duplicate of that.
